Client/Frontend/TabContentsScripts/CreditCardHelper.swift

```diff
@@ -213,4 +213,19 @@ class CreditCardHelper: TabContentScript {
                        category: .webview)
         }
     }
+
+    // Note: document.activeElement.blur() is used to remove the focus from the
+    // currently focused element on a web page. When an element is focused,
+    // it typically has a visual indication such as a highlighted border or change in appearance.
+    // The reason we do it is because after pressing done the focus still remains in WKWebview
+    static func blurActiveElement(tabWebView: WKWebView,
+                                  logger: Logger) {
+        let fillCreditCardInfoCallback = "document.activeElement.blur()"
```

Comment:
Synced with @issammani here and this is by far the best solution we have. I do see a tiny issue but nothing that is a showstopper and is more in terms of improvements on iPad and iFrame. 

Kudos @issammani 🚀

Reply:
I double checked an it works with iframes. Since `document.activeElement` will return:
- the `iframe` element itself when the input is inside the iframe.
- the `input` element otherwise.

So blurring will work consistently regardless of the context. Not sure what the issue we were seeing before was but it might just be a build issue 😄